psi4/driver/qcdb/cfour.py

```diff
@@ -80,7 +83,7 @@ def harvest_output(outtext):
     return pass_psivar[retindx], pass_coord[retindx], pass_grad[retindx]
 
 
-def harvest_outfile_pass(outtext):
+def local_harvest_outfile_pass(outtext):
```

Comment:
What's this function for, now?

Reply:
I don't understand your confusion. It takes the contents of a CFOUR output file as an argument and returns data structures containing variables (energies, gradients, etc.) scraped from the output file.

Admittedly, the docstring looks like somebody stopped partway through writing it.

Reply:
No, I mean, do we still need it, if we're using `qcng.programs.cfour.harvester.harvest_outfile_pass()`?

Reply:
You're right, it's dead code, as the new docstring admits. I get to delete the whole file soon, so I was avoiding deleting the fn piecemeal.